### PR TITLE
PROD 390: Run python modernize on the repo

### PR DESCRIPTION
**Pull request by jinder1s**

### CODE REVIEW

File: dependencies/shotgun_api3/lib/simplejson/decoder.py [hunk 1]
```diff
@@ -177,10 +180,11 @@ def py_scanstring(s, end, encoding=None, strict=True,
 WHITESPACE = re.compile(r'[ \t\n\r]*', FLAGS)
 WHITESPACE_STR = ' \t\n\r'
 
-def JSONObject((s, end), encoding, strict, scan_once, object_hook,
+def JSONObject(xxx_todo_changeme, encoding, strict, scan_once, object_hook,
```
Comment (azarembok) [inline dependencies/shotgun_api3/lib/simplejson/decoder.py]: Whats up with the variable name here?

File: dependencies/shotgun_api3/lib/simplejson/decoder.py [hunk 2]
```diff
@@ -269,7 +273,8 @@ def JSONObject((s, end), encoding, strict, scan_once, object_hook,
         pairs = object_hook(pairs)
     return pairs, end
 
-def JSONArray((s, end), scan_once, _w=WHITESPACE.match, _ws=WHITESPACE_STR):
+def JSONArray(xxx_todo_changeme1, scan_once, _w=WHITESPACE.match, _ws=WHITESPACE_STR):
```
Comment (azarembok) [inline dependencies/shotgun_api3/lib/simplejson/decoder.py]: and here?

File: dependencies/shotgun_api3/lib/simplejson/decoder.py (hunk 1 above)
Comment (jinder1s) [reply dependencies/shotgun_api3/lib/simplejson/decoder.py]: The modernizer is set to moved the tuple (s,end) to inside the function. The previous function definition is no longer valid in python 3

Comment (jinder1s) [reply dependencies/shotgun_api3/lib/simplejson/decoder.py]: I will change name to something else

File: dependencies/shotgun_api3/lib/simplejson/decoder.py (hunk 2 above)
Comment (jinder1s) [reply dependencies/shotgun_api3/lib/simplejson/decoder.py]: Tuple parameters are no longer support in Python3: http://www.python.org/dev/peps/pep-3113/

File: VEDA_OS01/management/commands/tests/test_create_oauth_client.py [hunk 3]
```diff
@@ -71,7 +73,7 @@ def assert_client_created(self, args, options):
             (g for g in GRANT_TYPES),
         ),
         # Generate all valid option combinations
-        (dict(zip(OPTIONAL_COMMAND_ARGS, p)) for p in product(
+        (dict(list(zip(OPTIONAL_COMMAND_ARGS, p))) for p in product(
```
[the remark this answers is not in the document]
Comment (jinder1s) [reply VEDA_OS01/management/commands/tests/test_create_oauth_client.py]: not sure myself, this was auto-modified by python modernizer

**Pull request merged**